docker/config-standalone.json

```diff
@@ -116,22 +116,20 @@
         }
       },
       {
-        "name": "SynchronizeBillingUsersTask",
-        "active": false,
-        "periodicity": "0 0 * * *",
-        "config": {}
-      },
-      {
-        "name": "SynchronizeBillingInvoicesTask",
+        "name": "BillingPeriodicOperationTask",
         "active": false,
-        "periodicity": "0 0 * * *",
-        "config": {}
+        "periodicity": "0 1 1 * *",
```

Comment:
This on seems not matching docker/config.json?
```
        "periodicity": "0 3 * * *",
        "config": {
          "onlyProcessUnpaidInvoices": false
        }
```
Will you update the batch server in AWS yourself?